src/cloud-api-adaptor/ibmcloud/ROKS_SETUP.md

```diff
@@ -0,0 +1,273 @@
+# PeerPod setup using Red Hat OpenShift on IBM Cloud
+
+This guide describes how to set up a simple peer pod demo environment with a Red Hat OpenShift Kubernetes cluster on IBM Cloud. This involves the following steps:
+
+1. [Set up an OpenShift Kubernetes cluster for PeerPod VMs](#set-up-an-openshift-kubernetes-cluster-for-peerpod-vms)
+1. [Upload a PeerPod VM Custom Image](#upload-a-peerpod-vm-custom-image)
+1. [Deploy the PeerPod Webhook](#deploy-the-peerpod-webhook)
+1. [Deploy the Confidential-containers operator](#deploy-the-confidential-containers-operator)
+1. [Run a Helloworld sample](#run-a-helloworld-sample)
```

Comment:
I see your point but I'll leave it in for now. I like to show an ordered list of steps to be clear, but can rethink this, along with other improvements, in a follow-up PR in Jan. If possible, I'd like to get some version of this doc in by tomorrow, because I'll be away after that until Jan 2, and I'd like to let others try it out while I'm gone.